DDDEastAnglia/DataAccess/SponsorSorter.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DDDEastAnglia.Models;
+
+namespace DDDEastAnglia.DataAccess
+{
+    public interface ISponsorSorter
+    {
+        IEnumerable<Sponsor> Sort(IEnumerable<Sponsor> sponsors);
+    }
+
+    public class SponsorSorter : ISponsorSorter
```

Comment:
Is this a `DescendingAmountSponsorSorter`? Or something similar? Perhaps `DefaultSponsorSorter` would be sufficient. Depends how likely we are to want to tweak the algorithm, etc.